web/src/components/njwds-extended/PrimaryButton.tsx

```diff
@@ -26,5 +29,14 @@ export const PrimaryButton = forwardRef(function PrimaryButton(
   props: Props,
   ref: Ref<HTMLButtonElement>
 ): ReactElement {
-  return <GenericButton {...props} className={colors[props.isColor]} ref={ref} />;
+  return (
+    <GenericButton
+      {...props}
+      className={`${colors[props.isColor]} 
+      ${props.isUnBolded ? "text-normal" : ""}  
+      ${props.isSmallerText ? "font-body-2xs" : ""} 
+      ${props.isLargeButton ? "usa-button--big" : ""}`}
+      ref={ref}
+    />
```

Comment:
I see that you moved these classes up a level so that they only exist via the PrimaryButton rather than for all generic buttons, 

what was the reasoning like for this? Is this better somehow? I would worry about these differences have to be undone if we ever want these attributes through other things like the secondary buttons because they seem quite common and plausible to have in other buttons later

Reply:
I'm going to send another PR. My goal is to exclude the props we don't use. I found a better solution.